### PR TITLE
Per project versioning

### PR DESCRIPTION
**Pull request by arkpar**

### CODE REVIEW

File: libdevcore/Common.cpp [hunk 1]
```diff
@@ -22,7 +22,7 @@
 #include "Common.h"
 #include "Exceptions.h"
 #include "Log.h"
-#include "BuildInfo.h"
+#include "dev/BuildInfo.h"
```
Comment (LefterisJP) [inline libdevcore/Common.cpp:25]: Won't work with a non-umbrella build. Fails on all platforms: http://52.28.164.97/job/project-build/31/

**Comment by arkpar** [comment]
https://github.com/ethereum/webthree-helpers/pull/18
https://github.com/ethereum/libweb3core/pull/6
https://github.com/ethereum/libethereum/pull/19
https://github.com/ethereum/mix/pull/75
https://github.com/ethereum/solidity/pull/79
https://github.com/ethereum/webthree/pull/41
https://github.com/ethereum/alethzero/pull/63
These PRs go together

**Comment by debris** [comment]
looksgood for umbrella

File: libdevcore/Common.cpp (hunk 1 above)
Comment (arkpar) [inline libdevcore/Common.cpp:25]: tested to work with split repos, just need other PRs merged

**Pull request merged**